src/nvim/search.c

```diff
@@ -1383,11 +1381,14 @@ int do_search(
         && c != FAIL
         && !shortmess(SHM_SEARCHCOUNT)
         && msgbuf != NULL) {
-      search_stat(dirc, &pos, show_top_bot_msg, msgbuf,
```

Comment:
`#define SEARCH_STAT_BUF_LEN 12` wasn't deleted on line 1045.

Reply:
Oops forgot it was moved to `search.h` . Deleted .